### ISSUE TITLE
Various NavigationProperties Missing

### ISSUE THREAD

**Issue by OptrixAU** [MaintenanceExperts/MEXODataAPI#1]
I'll update this listing with NavigationProperties that appear to not be present.

All of these would be extremely useful for **$expand** functionality, as most of them are many-to-one, meaning that there would be a common need to convert between the ID number and a human-readable meaning.

As it is, getting a human-readable output on some of these Entities would require a _minimum_ of 3 calls, instead of one. 

**Asset**: Manufacturer, Model Number, AssetStatus.
**Request**: Priority, Department, JobType, Asset

Basically, if a field-name ends in "ID" and it isn't the key for the entity, there should be a matching NavigationProperty to get its data. It would be great if this also included contacts (ie. CreatedContact, AcceptedContact, ModifiedContact). This would vastly reduce the number of calls to API functions that would be required, and also allows ODATA clients to have a proper, machine-readable understanding of your data.
